CartingApp/CartItem.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace CartingApp
+{
+    public class CartItem
+    {
+        public Product product;
+        public int quantity;
+        public  double totalCost;
```

Comment:
Public instance Variables like "totalCost" should be replaced with  public fields with private setters to improve security of code.
Will need modifications in code such that only the functions inside the class modify it.